src/containers/ViewerPanel/index.tsx

```diff
@@ -503,28 +599,67 @@ class ViewerPanel extends React.Component<ViewerPanelProps, ViewerPanelState> {
                             lastFrameTime={lastFrameTime}
                             loading={isBuffering}
                             isEmpty={status === ViewerStatus.Empty}
-                        />
-                        <RecordMoviesComponent
-                            movieUrl={this.state.movieURL}
-                            movieTitle={movieTitle}
-                            resetAfterMovieRecording={
-                                this.resetAfterMovieRecording
+                            minimalControls={
+                                this.embedDisplaySettings
+                                    .minimalPlaybackControls
                             }
-                            startRecording={simulariumController.startRecording}
-                            stopRecording={simulariumController.stopRecording}
+                            showHomeViewButton={
+                                this.embedDisplaySettings
+                                    .onlyRenderBottomControls
+                            }
+                            resetCamera={simulariumController.resetCamera}
                         />
+
+                        {!this.embedDisplaySettings.minimalPlaybackControls && (
+                            <RecordMoviesComponent
+                                movieUrl={this.state.movieURL}
+                                movieTitle={movieTitle}
+                                resetAfterMovieRecording={
+                                    this.resetAfterMovieRecording
+                                }
+                                startRecording={
+                                    simulariumController.startRecording
+                                }
+                                stopRecording={
+                                    simulariumController.stopRecording
+                                }
+                            />
+                        )}
+                        {location.pathname === EMBED_PATHNAME && (
+                            <ViewportButton
+                                tooltipText={
```

Comment:
The large amount of conditional rendering this changeset introduces, and the more complex logic behind it makes me think a `ControlsModule` that is a container and could access the redux store directly would be a useful refactor.

It could handle most of what is needed for rendering: `PlaybackControls`, `CameraControls`, `ScaleBar`, `RecordMoviesComponents`, and `FullScreenButton` and could reduce complexity in this file.

This changeset is already large though.